lib/queue_bus/publishing.rb

```diff
@@ -30,6 +30,7 @@ def publish_metadata(event_type, attributes = {})
       bus_attr = { 'bus_published_at' => Time.now.to_i, 'bus_event_type' => event_type }
       bus_attr['bus_id']           = "#{Time.now.to_i}-#{generate_uuid}"
       bus_attr['bus_app_hostname'] = ::QueueBus.hostname
+      bus_attr['bus_context'] = ::QueueBus.context unless ::QueueBus.context.nil?
```

Comment:
Can we please also get a test for this metadata as well?

Reply:
yes! good catch!